packages/docs/docs/docs/erc20_onboarding.md

```diff
@@ -27,17 +27,17 @@ This on-boarding plan considers the following assumptions:
 
 ## Onboarding plan demo
 
-The central idea of this proposal is to deploy an upgradeable version of your token, using the ZeppelinOS command line
-tool. Additionally, we will use the [`ERC20Migrator`](https://github.com/OpenZeppelin/openzeppelin-eth/blob/v2.0.0/contracts/drafts/ERC20Migrator.sol)
-contract provided by `openzeppelin-eth`, the [OpenZeppelin EVM package for ZeppelinOS](https://github.com/OpenZeppelin/openzeppelin-eth/).
+The central idea of this proposal is to deploy an upgradeable version of your token, using the OpenZeppelin command line
```

Comment:
Well, the CLI **is** a tool for using OZ-Contracts (as well as other contracts). Given that we won't be making a CLI specific to OZ-contracts, I'd just name it _OZ CLI_ instead of _OZ SDK CLI_, so it sounds less heavy.